content/blog/multitenancy-for-kubernetes-clusters-using-hpe-alletra-6000-and-nimble-storage.md

```diff
@@ -0,0 +1,234 @@
+
+Example.
+
+```yaml
+kind: Pod
+apiVersion: v1
+metadata:
+  name: my-csi-app-inline-volume
+spec:
+  containers:
+    - name: my-frontend
+      image: busybox
+      command: [ "sleep", "100000" ]
+      volumeMounts:
+      - mountPath: "/data"
+        name: my-csi-volume
+  volumes:
+  - name: my-csi-volume
+    csi:
+      driver: csi.hpe.com
+      nodePublishSecretRef:
+        name: my-tenant-secret
+      fsType: ext3
+      volumeAttributes:
+        csi.storage.k8s.io/ephemeral: "true"
+        accessProtocol: "iscsi"
+        size: "5Gi"
+```
+
+The `Secret` "my-tenant-secret" would have to exist in the same `Namespace` as the `Pod` and contain the necessary connectivity and credentials to the tenant.
+
+## Virtualization and Containers
+
+Deploying Kubernetes on a virtualization platform such as VMware vSphere, OpenStack or Hyper-V is by far the most popular pattern for deploying on-premises Kubernetes. Many times customers want to leverage the same array to provide persistent storage both for the virtualization platform and the container platform.
+
+Allowing Kubernetes clusters "administrator" or "poweruser" access to the array served by the virtualization platform the cluster is running on might be feasible in a single tenant and single application type scenario. Once weaving in the Ephemeral Inline Volumes use case into the mix and we've basically given application administrators way too many privileges on the array.
+
+In many cases the virtualization and storage administrator is combined into the same role, moving forward, this administrative function would be able to securely hand over credentials to Kubernetes administrators that needs a first class persistent storage solution.
+
+## Kubernetes-as-a-Service
+
+Cloud and Managed Service Providers (CSPs and MSPs) monetizing their infrastructure are in a constant battle to safely and securely share infrastructure resources between their tenants and at the same time provide a differentiating portfolio. In the case of dispensing Kubernetes clusters to their tenants they would have to resort to either using the virtualization platform CSI driver (such as the vSphere CSI driver) which is incredibly limited in functionality or running a Container Attached Storage (CAS) solution on the Kubernetes cluster itself which in turn would result in storage and performance inefficiencies.
+
+With multitenancy, CSPs and MSPs are now enabled to create new tenants on the array as part of the their catalog workflows and provide an entirely new set of rich data services enabled by HPE Alletra 6000 and HPE Nimble Storage.
+
+![Kubernetes as a Service](/img/kaas.png "Kubernetes as a Service")
+
+# Summary
+
+Expect more content that elaborates deeper on how multitenancy can be used with Kubernetes using HPE Alletra 6000 and Nimble Storage. Consider this blog post a teaser of the cornerstone capability of multitenancy.
```

Comment:
Take out the word "deeper" so it reads "Expect more content that elaborates on how multitenancy..."
Second sentence, change "this blog post a teaser of the cornerstone" to "this blog post a teaser regarding the cornerstone"